### DISCUSSION TITLE
Labelling is completely and utterly messed up

### DISCUSSION THREAD

**Discussion by naahedd** [ultralytics/ultralytics#15377] [Q&A]
![image](https://github.com/user-attachments/assets/51f4dc8b-9c35-4f5b-a36c-c7f09756d7dd)
Hey all, hope all is well.
I think I need some serious help here and I wish someone could help me with that.
I have tried many things but I'm getting the same results.

**Comment by pderrenger** [comment]
Hi, thanks for reaching out. Please ensure you're using the latest version of the Ultralytics package. If the issue persists, could you provide more details about your setup and the steps you've taken? This will help us diagnose the problem more effectively.
